src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -218,12 +218,19 @@
   <PropertyGroup>
     <GetNuGetPackageVersionsDependsOn>$(GetNuGetPackageVersionsDependsOn);CreateVersionInfoFile</GetNuGetPackageVersionsDependsOn>
   </PropertyGroup>
+    
+  <PropertyGroup>
+    <SyncInfoDirectory Condition="'$(SyncInfoDirectory)' == ''">$(ObjDir)</SyncInfoDirectory>
+    <SyncInfoFile Condition="'$(SyncInfoFile)' == ''">$(SyncInfoDirectory)version.txt</SyncInfoFile>
```

Comment:
agreed, but it will be an issue for local builds, although I guess people might not care if the binaries that they build might have an inaccurate version